lib/scrabble.rb

```diff
@@ -1,7 +1,59 @@
+require 'pry'
+
 class Scrabble
 
   def score(word)
-    1
+    if word == "" || word == nil
+      points = 0
+    else
+      word_array = word.upcase.chars
+      points = 0
+      word_array.map! do |char|
+        char = point_values[char]
+      end
+      word_array.each do |number|
```

Comment:
Having each do something to a collector object, is a sign that there is a better choice enumerable.